### PR TITLE
GMO PostPay icon

### PR DESCRIPTION
**Pull request by KentaroHagi**
** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [ ] I have provided a link to the brand icon’s brand guidelines whenever possible.

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

File: app/assets/images/payment_icons/gmo-postpay.svg [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" aria-labelledby="pi-gmo-postpay">
+  <title id="pi-gmo-postpay">GMO後払い</title>
```
Comment (larouxn) [inline app/assets/images/payment_icons/gmo-postpay.svg:2]: We need to make the `title` here and the `label` in `payment_icons.yml` the same value.

File: app/assets/images/payment_icons/gmo-postpay.svg [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      <path fill="#005bac" d="M31.08,11.92l.3,1.3a1.69,1.69,0,0,1-1,.26h-.22l.66-3.64h-.7l-.63,3.64h-.27v.62h1.13a2.14,2.14,0,0,0,1.14-.27l.07.33h.63l-.47-2.24Z"/>
+      <path fill="#005bac" d="M29.42,12.4v-.62l-.05,0-.53.16V11h.58v-.57h-.58V9.84h-.67v.55h-.7V11h.7V12.1l-.67.08h0v.6h0l.66-.09v.75a.15.15,0,0,1-.16.13H27.6v.59h.62c.39,0,.62-.21.62-.55v-1c.23-.05.4-.1.56-.15Z"/>
+    </g>
+    <g>
+      <path fill="#005bac" d="M10.83,13.82h.28s0,0,0,0l1.17-1.91L13,14.07l0,0h1.22s0,0,0,0L12.94,10l0,0h-.46l0,0-1.58,2.44L9.26,10l0,0H8.76l0,0-1.3,4.09,0,0H8.68l0,0,.63-2.19,1.17,1.91a.05.05,0,0,0,0,0Z"/>
+      <path fill="#005bac" d="M7.49,11.78h0s0,0,0,0H4a.34.34,0,1,0,0,.67H6.05A2.16,2.16,0,0,1,4,13.56c-1.19,0-2.15-.69-2.15-1.54s1-1.53,2.15-1.53A2.62,2.62,0,0,1,5.6,11H7a0,0,0,0,0,0,0h0A4.35,4.35,0,0,0,4,9.9C2,9.9.44,10.85.44,12S2,14.15,4,14.15s3.53-1,3.53-2.13A1,1,0,0,0,7.49,11.78Z"/>
+      <path fill="#005bac" d="M17.69,9.9c-2,0-3.54.95-3.54,2.12s1.58,2.13,3.54,2.13,3.54-1,3.54-2.13S19.64,9.9,17.69,9.9Zm0,3.66c-1.19,0-2.15-.69-2.15-1.54s1-1.53,2.15-1.53,2.15.69,2.15,1.53S18.87,13.56,17.69,13.56Z"/>
+    </g>
+  </g>
+</svg>
```
Comment (larouxn) [inline app/assets/images/payment_icons/gmo-postpay.svg:22]: Can we optimize this SVG with `svgo` according to the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/8cc69fcbdf8c73e0ccef9010982587227825bf4f/CONTRIBUTING.md)?

```
npm install -g svgo@1.3.2
svgo app/assets/images/payment_icons/gmo-postpay.svg --disable={removeUnknownsAndDefaults,removeTitle,cleanupIDs,removeViewBox}
```

**Review by larouxn — COMMENTED** [review]
👋 Thanks for the contribution @KentaroHagi. I left two small comments. One about the `title` and one about SVG optimization. ([CI logs](https://github.com/activemerchant/payment_icons/runs/4379970112?check_suite_focus=true))

Tip, you can run the CI tests locally with `bundle exec rails test`.

**Comment by KentaroHagi** [comment]
Thank you for your advice, @larouxn .
I modified the gmo-postpay.svg file, and sent pull request.
Could you check the request again?

**Review by larouxn — COMMENTED** [review]
One more comment about SVG code.

The real issue now I see is the logo seems to be to large, it's coming out of the background, border. Can we look into making it small so it fits inside like the Visa example here?
Screenshot 2021-12-02 190054

**Comment by KentaroHagi** [comment]
Thank you for pointing this out @larouxn . I have corrected icon size and `svg` tag.
I would appreciate it if you could check pull request.

**Review by tauthomas01 — APPROVED** [review]

**Review by larouxn — APPROVED** [review]
LGTM! :clap: 
Screenshot from 2021-12-14 18-53-02

**Comment by larouxn** [comment]
👋 @KentaroHagi, your icon should be available everywhere now. Have a great day! ☀️

**Pull request merged**